packages/stark-core/src/modules/error-handling/error-handling.module.ts

```diff
@@ -6,8 +6,8 @@ export class StarkErrorHandlingModule {
 	/**
 	 * Instantiates the services only once since they should be singletons
 	 * so the forRoot() should be called only by the AppModule
-	 * @link https://angular.io/guide/singleton-services#forroot
-	 * @returns a module with providers
+	 * @link https://v7.angular.io/guide/singleton-services#the-forroot-pattern
```

Comment:
Actually this `@link` is incorrect according to the doc of compodoc :disappointed: 
And so it is not displayed in the generated doc

![image](https://user-images.githubusercontent.com/10333565/77832544-cec9fe00-712e-11ea-8001-0c485f9d6bcd.png)


See [Compodoc JSDoc Tags](https://compodoc.app/guides/jsdoc-tags.html):

```
@link : you can use these three syntax like JSDoc:

//for an internal reference

{@link Todo}
[Todo]{@link Todo}
{@link Todo|TodoClass}

Anchors are supported : [Todo]{@link Todo#myproperty}

//for an external link

[Google]{@link http://www.google.com}
{@link http://www.apple.com|Apple}
{@link https://github.com GitHub}
```

That means we should change it everywhere... Because no link is displayed anywhere :confused: 

If you want, I can update your PR with the fix for those `@link` ! :blush:

Reply:
Ah indeed, I fixed most of those links (in `stark-core` only) but you're right I forgot some of them.
No worries I will fix those links too because in any case I still have to add to this PR the commits for the docs of `stark-ui` and `stark-rbac` 😉

Reply:
@SuperITMan I've added a new commit with the fixes for the `@link`s in the `stark-core` docs...
That way you can check if I didn't forget something 😉 (I'll squash this when I finish the changes and add the commits for the other packages).